src/components/create/NumberOfAttempts.tsx

```diff
@@ -72,7 +72,7 @@ const NumberOfAttempts = ({ initAttempts, onChange }: Props) => {
   };
 
   return (
-    <FormControl sx={{ mt: 3, width: '25ch' }} variant="outlined">
+    <FormControl sx={{ width: '25ch' }} variant="outlined">
```

Comment:
Can this width be something like `min-content` ? if not possible it is fine to stay `25ch`.

Reply:
With min-content, I take too few space, the text is hidden. So I think I will stay with 25ch.